aot-core/src/test/groovy/io/micronaut/aot/core/codegen/ApplicationContextConfigurerGeneratorTest.groovy

```diff
@@ -78,6 +101,22 @@ public class AOTApplicationContextConfigurer implements ApplicationContextConfig
 
   private static void initializer2() {
   }
+
+  private static List list0() {
+    List result = new ArrayList<>(2);
+    result.add("static-init");
+    result.add("static-init");
```

Comment:
Not really important but is this duplication correct? Wouldn't it be better to use a set?

Reply:
The duplication is simply because it's a test optimization added twice, which would normally not happen.